### PR TITLE
Pre Codefreeze For Demo

### PR DESCRIPTION
**Pull request by iwantmyhatback**
Styled Components for Demonstration, and got static data to display reliably

### CODE REVIEW

File: src/components/questions/answer.js [hunk 1]
```diff
@@ -8,7 +9,17 @@ constructor(props){
...
-    <div>Answer</div>
+    <div className="tile is-child box">
+      <div className="title is-5">{`A: ${this.props.body}`}</div>
+      <div>{`by ${this.props.answerer_name}, ${formatDate(this.props.date)}`}</div>
+      <div>| Helpfulness:{" "}{this.props.helpfulness}</div>
+      {this.props.photos.map( (photo, index)=>(<img
+      style={{maxHeight: 100, maxWidth: 100}}
+      src={photo}
+      alt="new"
+      />))}
```
Comment (RyanTFulmer) [inline src/components/questions/answer.js:20]: nice styling start!!

File: src/components/questions/answersList.js [hunk 2]
```diff
@@ -10,13 +10,18 @@ class AnswerList extends React.Component{
...
-    <div>
       <div>
-        AnswerList (map answers to Answerlist)
-      </div>
-      <Answer/>
+      {
+      Object.keys(this.props.answers)[0] ? (Object.keys(this.props.answers).map( (key, index) => {
+        return <Answer key={index} id={this.props.answers[key].id} body={this.props.answers[key].body} date={this.props.answers[key].date} answerer_name= {this.props.answers[key].answerer_name} helpfulness={this.props.answers[key].helpfulness} photos={this.props.answers[key].photos}/>
+      })) : <div>{" "}</div>
+      }
```
Comment (RyanTFulmer) [inline src/components/questions/answersList.js:24]: if this is just a list of keys consider using a set

File: src/formatDate.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  } else if (month===6){
+    month = "June"
+  } else if (month===7){
+    month = "July"
+  } else if (month===8){
+    month = "August"
+  } else if (month===9){
+    month = "September"
+  } else if (month===10){
+    month = "October"
```
Comment (RyanTFulmer) [inline src/formatDate.js:26]: there's gotta be an easier way to do this! But if it works, it ships :)

**Review by RyanTFulmer — APPROVED** [review]
Hey Tristan! Things look great, nice to see your styling underway. Added some comments to help optimize later

File: src/formatDate.js (hunk 3 above)
Comment (iwantmyhatback) [reply src/formatDate.js:26]: thats my life motto!

File: src/components/questions/answersList.js (hunk 2 above)
Comment (iwantmyhatback) [reply src/components/questions/answersList.js:24]: its actually just a work around for using map with an object, i will comment in whats going on when we're not on this time crunch for our upcoming demo

**Pull request merged**